account_payment_mode/models/account_payment_method.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
```

Comment:
I think the '_' is missing

Reply:
Is not necessary for _sql_constraint messages